File: business/src/main/kotlin/com/isystk/sample/common/dto/DefaultPageable.java
package com.isystk.sample.common.dto;

public class DefaultPageable implements Pageable {

  int page = 1;

  int perpage = 10;

  public DefaultPageable(int page, int perpage) {
    this.page = page;
    this.perpage = perpage;
  }

  public int page() {
    return this.page;
  }

  public int perpage() {
    return this.perpage;
  }

}

File: business/src/main/kotlin/com/isystk/sample/common/dto/DefaultPageable.kt
package com.isystk.sample.common.dto

class DefaultPageable(page: Int, perpage: Int) : Pageable {
    var page = 1
    var perpage = 10

    init {
        this.page = page
        this.perpage = perpage
    }

    override fun page(): Int {
        return page
    }

    override fun perpage(): Int {
        return perpage
    }
}